### ISSUE TITLE
[Request] Alphabet Strip and Next Aired

### ISSUE THREAD

**Issue by Blurayx** [jurialmunkey/skin.eminence.2#92]
For a change of scenery, I decided to go back to one of my old favorites. I can't remember does eminence have an alphabet strip option and will you be implementing  Next Aired(Library) Common shortcut? 

Thanks

**Comment by jurialmunkey** [comment]
No it doesn't have alphabet strip - I think the alphabet strip didn't really exist at the time. Definitely something that can be added as a feature though!

Next Aired implementation using TMDbHelper (similar to how Arctic Horizon does it) is coming soon!

PS - I'm also working on Arctic Horizon 2 -- that's why Arctic Horizon hasn't had updates in a while (plus just been busy with work this year). It's being built from the ground up with a focus on mouse/touch support so it's still a while off yet though!

**Comment by Blurayx** [comment]
Ah, great news regarding eminence I will be using it on the Apple TV 4k(2021) because navigation is very smooth & slick.

Use Arctic Horizon on the Nvidia Shield TV.
Wow, Arctic Horizon 2. There's no stopping you. At one point I thought you were going to take on Aeon 2, it had so much potential.

**Comment by derrgoo** [comment]
Both skins are top notch! Eminence just run a so slick on small devices. Great to be back!

**Comment by jurialmunkey** [comment]
>Ah, great news regarding eminence I will be using it on the Apple TV 4k(2021) because navigation is very smooth & slick. Using Arctic Horizon on the Nvidia Shield TV.
>Both skins are top notch! Eminence just runs so slick on small devices. 

Yeah that's sort of the plan moving forward - Eminence as my lighter skin that'll run well on practically anything and has the classic approach to the home-screen; then Arctic Horizon as my more fully featured multi-widget skin for devices with a bit more power. 

The AH rewrite should improve performance a fair bit to make it (hopefully) viable on some lower powered devices but Eminence will always have the edge performance wise due to its simplicity.

**Comment by Blurayx** [comment]
That was Quick. I wasn't expecting you to add an alphabet strip until the end of the year!!!!

Thank You 😀
